Classes/Domain/Dto/CurrentPage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\Deepltranslate\Core\Domain\Dto;
+
+use WebVision\Deepltranslate\Core\Exception\MissingParameterException;
+
+final class CurrentPage
```

Comment:
Please add some class PHP docblock comment describing what this class is for.
The comment should also state if this is internal or when considered public api emphasize this and give a brief explanation for the usage (for custom addon extensons ?).